numpy/random/tests/test_generator_mt19937.py

```diff
@@ -1876,7 +1876,7 @@ def test_hypergeometric(self):
         bad_nsample_one = [-1]
         bad_nsample_two = [4]
         hypergeom = random.hypergeometric
-        desired = np.array([1, 1, 1])
+        desired = np.array([0, 0, 1])
```

Comment:
Why does the result change?

Reply:
(See my earlier response to the same question about the other test result change.)